lib/react_client/react_interop.dart

```diff
@@ -178,8 +192,7 @@ class InteropProps {
   external factory InteropProps({ReactDartComponentInternal internal, String key, dynamic ref});
 }
 
-/// Internal react-dart information used to proxy React JS lifecycle to Dart
-/// [Component] instances.
+/// A Dart object that stores .
```

Comment:
Looks like this was leftover from my one of my commits, sorry 😬. Can you revert this doc comment change, please?